microsoft/testsuites/asr/cvt.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import base64
+import datetime
+import re
+import uuid
+from pathlib import Path
+from typing import Any, Dict
+
+from assertpy import assert_that
+
+from lisa import (
+    CustomScript,
+    CustomScriptBuilder,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    create_timer,
+    schema,
+    simple_requirement,
+)
+from lisa.executable import ExecutableResult
+from lisa.features import Disk
+from lisa.sut_orchestrator.azure.features import AzureExtension
+from lisa.tools import Find, Mkdir, Uname, Wget
+
+
+@TestSuiteMetadata(
+    area="cvt",
+    category="functional",
```

Comment:
area="vm_extension",
    category="functional",

Reply:
CVT test validates the functionality of ASR driver.
The driver is installed on the system using extension.
The test is performed post installing the extensions